packages/test/test-end-to-end-tests/src/test/gc/gcVersionUpdate.spec.ts

```diff
@@ -98,14 +100,27 @@ describeCompat("GC version update", "NoCompat", (getTestObjectProvider, apis) =>
 		return summaryResult.summaryVersion;
 	}
 
+	/**
+	 * Generates a summary and validates that the GC state is reset (empty) in the base GC details. This will ensure
```

Comment:
> in the base GC details

This is about the snapshot loaded from, not the result of the summary here, right? The summarize just serves to ensure GC is initailized?

Reply:
Yes, updated the comment to make that clear.